examples/console/README.md

```diff
@@ -0,0 +1,28 @@
+# Console Example
+
+This module contains an example using the SDK in this repository that supports a
+single payment channel controlled by two people through a command line interface
+using simple TCP network connection and JSON messages.
+
+## State
+
+This example is a rough example and is brittle and is still a work in progress.
+
+## Usage
+
+Follow the [Getting Started](../../Getting%20Started.md) instructions to run a
+stellar-core and horizon with CAP-21 capabilities.
+
+Run the example providing a key and its secret key.
+
+```
+git clone https://github.com/stellar/experimental-payment-channels
+cd examples/console
+go run . -horizon=http://localhost:8000 -account=G... -signer=S...
```

Comment:
it looks like the way it's set up actual keys need to be passed

Just use default random pub/secret keys?
```suggestion
go run . -horizon=http://localhost:8000 -account= GCIFBFIALD3GDBNHAA7QHOYMYFEXVZTRY6ICHWDDB5HELJRGVJZ4HB5V -signer= SDD7AE7ZFFXWC4LNAILC5XQCV7J5BHGUJULB2MAHQ4SQS5SCUYG2YFUW
```